.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required
+
+language: sh
+
+services:
+  - docker
+
+before_script:
+  - docker pull docker.io/koalaman/shellcheck
+
+script:
+  # shellcheck validate
+  - docker run -v "$PWD:/mnt:ro" docker.io/koalaman/shellcheck -x $(find . -path ./mantle -prune -and -path ./ostree-releng-scripts -prune -o -type f -exec grep -l '^#!.*bash' {} \;)
+  # build dockerfile
+  - docker build -t coreos-assembler -f ./Dockerfile .
```

Comment:
That's...a currently very heavyweight operation that will flake for sure.  I feel like we should tackle CI for the container build separately.